butterknife/src/main/java/butterknife/ButterKnife.java

```diff
@@ -83,7 +85,7 @@ private ButterKnife() {
     throw new AssertionError("No instances.");
   }
 
-  /** An unbinder contract that can be bind with {@link butterknife.Unbinder}. */
+  /** An unbinder contract that will unbind views when called. */
   @SuppressWarnings("unused") // Used by generated code.
   public interface ViewUnbinder<T> {
```

Comment:
I had the same thought. I'm not totally familiar with the original reasoning for it, but I think it's unfortunately necessary. In particular, this seems to be necessary in order to allow proper overriding of the `void unbind(T target)` method in subclassed `Unbinder` classes. Otherwise, if you have type `C$$ViewBinder.Unbinder`, and later have `G$$ViewBinder.Unbinder extends C$$ViewBinder.Unbinder`, then the subclass will still expect a type `C` in its `unbind()` method. Not sure if there's some generics-fu I'm missing here that could solve this.